extra/alacritty.man

```diff
@@ -39,8 +39,8 @@ Increases the level of verbosity (the max level is \fB\-vvv\fR)
 Prints version information
 .SH "OPTIONS"
 .TP
-\fB\-\-class\fR <class>
-Defines the window class on Linux [default: Alacritty]
+\fB\-\-class\fR [ <name> | <name>,<class> ]
```

Comment:
This should be named with `[ <instance> | <instance>,<general> ]` to follow the rest of the naming conventions.